.circleci/config.yml

```diff
@@ -20,36 +20,42 @@ jobs:
           name: Write Job name
           command: echo $CIRCLE_JOB > circlejob.txt
 
-      - restore_cache:
-          key: deps1-{{ .Branch }}-{{ checksum "dev-requirements.txt" }}-{{ checksum "package.json" }}-{{ checksum ".circleci/config.yml" }}-{{ checksum "circlejob.txt" }}
-
       - run:
           name: Install dependencies
           command: |
-              sudo pip install virtualenv --upgrade
+              sudo pip install --upgrade virtualenv
               python -m venv venv || virtualenv venv
               . venv/bin/activate
-              pip install -e .
-              pip install -r dev-requirements.txt --quiet
-              pip list | grep dash
+              pip install -r requirements-base.txt --quiet
               npm install --ignore-scripts
 
-      - run:
-          name: Build
-          command: npm run build
-
-      - save_cache:
-          key: deps1-{{ .Branch }}-{{ checksum "dev-requirements.txt" }}-{{ checksum "package.json" }}-{{ checksum ".circleci/config.yml" }}-{{ checksum "circlejob.txt" }}
-          paths:
-              - "venv"
-              - "node_modules"
-
       - run:
           name: Lint & format
           command: |
               npm run lint:test
               npm run format:test
 
+      - run:
+          name: Install dependencies (transient)
+          command: |
+              . venv/bin/activate
+              git clone git@github.com:plotly/dash.git
+              pip install -e ./dash --quiet
+              git clone git@github.com:plotly/dash-core-components.git
+              cd dash-core-components && npm i && cd ..
+              cd dash-core-components && npm run build && cd ..
+              pip uninstall dash-core-components -y
+              pip install -e ./dash-core-components --quiet
+
+      - run:
+          name: Build
+          command: |
+              . venv/bin/activate
+              npm run build
+              pip uninstall dash-renderer -y
+              pip install -e . --quiet
+              pip list | grep dash
```

Comment:
Build & install dash-renderer, print the installed dash packages, for sanity in case of problems..